ModernWallet/Classes/ViewController/CashOut/CashOurBankAccountDetailsViewController.swift

```diff
@@ -56,6 +75,18 @@ class CashOurBankAccountDetailsViewController: UIViewController {
         (currencyTextField.rx.textInput <-> bankDetailsViewModel.accountHolderAddress)
             .disposed(by: disposeBag)
         
+        (accountHolderCountryTextField.rx.textInput <-> bankDetailsViewModel.accountHolderCountry)
```

Comment:
Please move all bindings in dedicated ViewModel extension, same as in TransactionsStep1ViewController